pages/about/groups/task-forces/pdf-aam.md

```diff
@@ -0,0 +1,98 @@
+
+Some information on this page is also shown on, and may be more current in, the [PDF-AAM page](https://www.w3.org/groups/tf/pdf-aam/).
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::options toc_levels="2..3" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" %}
+{:/}
+
+-   TOC is created automatically.
+{:toc}
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Objective
+
+The mission of the PDF-AAM Task Force is to specify how user agents should map accessibility semantics within the Portable Document Format specification. This Task Force provides a focused forum for this work while the Working Group continues its work on ARIA development.
+
+### Work Statement
+
+The [PDF-AAM Task Force Work Statement](/about/groups/task-forces/pdf-aam/work-statement/) defines the initial objective, scope, approach, and participation of the Task Force.
+
+## Meetings and Communication
+
+The PDF-AAM Task Force conducts its work using a variety of synchronous and asynchronous tools.
+
+- Teleconferences of the ARIA Working Group and its task forces (also see [meeting minutes](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes))
+- Face-to-face meetings
+- [Mailing lists](#mailing-lists)
+- IRC (Internet Relay Chat) discussion on the [#pdf-aam](irc://irc.w3.org/pdf-aam) IRC channel, used largely for minute-taking
+- [PDF-AAM source repository issue tracker](https://github.com/w3c/pdf-aam/issues)
+- [ARIA specifications repository](https://github.com/w3c/aria)
+- [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/159734/)
+
+These tools are used by the Task Force participants. For ways that non-participants can contribute to the ARIA Working Group, including filing comments on publications, see [ARIA Contribution](/about/groups/ariawg/contribute/).
+
+### Teleconferences
+
+The PDF-AAM Task Force meets the third Wednesday of the month, 1:00–1:30 PM Eastern Time. For more details, including the meeting time in local time zones, see [PDF-AAM Calendar](https://www.w3.org/groups/tf/pdf-aam/calendar).
+
+### Meeting Minutes
+
+[Minutes from previous meetings](https://www.w3.org/WAI/ARIA/task-forces/pdf-aam/minutes) are available.
+
+### Mailing Lists
+
+The PDF-AAM Task Force uses the following mailing list for email discussion: [public-pdf-aam@w3.org Mail Archives](http://lists.w3.org/Archives/Public/public-pdf-aam/). 
+
+Participants are automatically added to the mailing list when they become a participant of the Task Force.
+
+## Current Work
+
+For the current PDF Accessibility API mappings, see the [Editor’s Draft](https://w3c.github.io/pdf-aam/).
+
+## Publications
+
+The PDF-AAM Task Force is primarily developing [PDF Accessibility API Mappings 1.0](https://w3c.github.io/pdf-aam/). 
+
+See also the [PDF-AAM GitHub issue tracking repository](https://github.com/w3c/pdf-aam/).
```

Comment:
Is this a publication? 
Suggest removing as it duplicates information on the page.

Reply:
> Suggest removing as it duplicates information on the page.

I am assuming you mean removing

> Suggest removing as it duplicates information on the page.

Removed.